### PR TITLE
608 school news and events

### PR DESCRIPTION
**Pull request by kevinhowbrook**
*contains* https://github.com/torchbox/rca-wagtail-2019/pull/511

Ticket [608](https://projects.torchbox.com/projects/rca-website-rebuild/tickets/608)

This pull request:
 - Enables News and Event listing by tag to the legacy site. 
 - Defines a carousel to be used for student stories

### CODE REVIEW

**Comment by richbrennan** [comment]
This PR is huge (due to other unmerged PRs and has conflicts, I think some dependent work needs to be merged and the conflicts resolved before it can be reviewed.

File: rca/schools/models.py [hunk 1]
```diff
@@ -241,6 +275,11 @@ class SchoolPage(BasePage):
         ),
         InlinePanel("stats_block", label="Statistics", max_num=1),
     ]
+    news_and_events_panels = [
+        FieldPanel("news_and_events_heading"),
+        InlinePanel("student_stories", label="Student Stories"),
```
Comment (kevinhowbrook) [inline rca/schools/models.py:280]: Needs limiting to 1

**Comment by kevinhowbrook** [comment]
> This PR is huge (due to other unmerged PRs) and has conflicts, I think some dependent work needs to be merged and the conflicts resolved before it can be reviewed.

Thanks @richbrennan, I rebased this now.

File: rca/schools/models.py [hunk 2]
```diff
@@ -408,6 +427,14 @@ def get_student_research(self, student_research, request):
             "slides": related_list_block_slideshow(student_research.slides),
         }
 
+    def get_student_stories(self, student_stories, request):
+        if not student_stories:
+            return
```
Comment (richbrennan) [inline rca/schools/models.py:432]: Nitpick: be be explicit in what's being returned (`None`/`{}`...)

**Review by richbrennan — COMMENTED** [review]
One nitpick, thanks Kevin

**Pull request merged**